backend/scripts/add_user.py

```diff
@@ -0,0 +1,16 @@
+from argparse import ArgumentParser
```

Comment:
Since this is a script, I guess it’s better to add `#!/usr/bin/python3` or `#!/usr/bin/env python3`

Reply:
Same in all the following scripts